todolist.js

```diff
@@ -50,9 +60,18 @@
     displayTasks(taskData);
     // }
   }
+
   const dispBtn = document.querySelector('.displayTasks');
   dispBtn.addEventListener('click', getDataFromLocalStorage);
 
+  const clearLocalStorage = document.querySelector('.clearLocalStorage');
+  clearLocalStorage.addEventListener('click', clrLocalStorage);
+
+  function clrLocalStorage() {
+    localStorage.clear();
+    location.reload(true);
```

Comment:
don't use reload() method, because it will take more time to load again the dom

Reply:
An alternative to this has been implemented by removing the displayed data.